combat-trainer.lic

```diff
@@ -27,11 +27,23 @@ class SetupProcess
     @last_cycle_time = Time.now - @cycle_armors_time
     @combat_training_abilities_target = settings.combat_training_abilities_target
     echo("  @combat_training_abilities_target: #{@combat_training_abilities_target}") if $debug_mode_ct
-
+    @cycle_regalia = settings.cycle_armors_regalia
+    echo("  @cycle_regalia: #{@cycle_regalia}") if $debug_mode_ct
+    @default_armor = settings.default_armor_type
```

Comment:
❓ Where is `default_armor_type` used?

Reply:
Thanks for the reviews!

default_armor_type was already called in armor_hysteresis when I started looking at combat-trainer.  To my eyes its intent was to define a skill for the NORMAL armor cycling functions that it would revert to when all the armors had been trained to intended levels.  Eg. once all four skills are above the threshold in armor_hysteresis it swaps to your previously defined 'light armor' set or whatever.

HOWEVER, although the logic was already in place it was not initialized.  There was no declaration for @default_armor = settings.default_armor_type or anything, so it didn't do anything, and as a result I don't think armor_hysteresis was working exactly as intended.  I added the missing declaration into the initialize portion, added it to base.yaml, and tested it.  Seems to work for normal armor swapping and definitely works for regalia.  My default armor type is Brigandine, which I have crafting techs for.  So once it trains all four up to max it then swaps to my brig regalia set and squats on it until one of the others drop below the training threshold.

I have no idea why it wasn't declared when I got there.  I suspect it was just a little mistake, since it seems to work just fine now that I've turned it on.  If nobody declares it then armor_hysteresis will continue doing what it did before.

Reply:
That makes sense, thank you